example/src/main/java/com/stripe/example/activity/LauncherActivity.kt

```diff
@@ -36,6 +36,14 @@ class LauncherActivity : AppCompatActivity() {
         private val activity: Activity
     ) : RecyclerView.Adapter<ExamplesAdapter.ExamplesViewHolder>() {
         private val items = listOf(
+            Item(
+                "Launch PaymentSheet Complete",
+                LaunchPaymentSheetCompleteActivity::class.java
+            ),
+            Item(
+                "Launch PaymentSheet Custom",
+                LaunchPaymentSheetCustomActivity::class.java
+            ),
```

Comment:
Are Complete and Custom the correct names?

Reply:
I created something to track this.